boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_defconfig

```diff
@@ -0,0 +1,22 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF53X=y
+CONFIG_SOC_NRF5340_CPUAPP_QKAA=y
+CONFIG_BOARD_NRF5340_AUDIO_DK_NRF5340_CPUAPP=y
+
+CONFIG_ARM_MPU=y
+
+CONFIG_HW_STACK_PROTECTION=y
+
+CONFIG_ARM_TRUSTZONE_M=y
+
+CONFIG_GPIO=y
+
+CONFIG_SERIAL=y
+
```

Comment:
Comments from original file seem to have been removed?

Reply:
Yes. I can add them back if required.
I dont feel comments on this format
`# Enable hardware stack protection
CONFIG_HW_STACK_PROTECTION=y

# Enable TrustZone-M
CONFIG_ARM_TRUSTZONE_M=y

# enable GPIO
CONFIG_GPIO=y`

Adds value. These configs are easily searchable in the Zephyr docs.